setup.py

```diff
@@ -347,7 +347,7 @@ def finalize_options(self):
         'numpy >= 1.11.3',
         'scipy >= 0.18.1',
         'six >= 1.5.0',
-        'smart_open >= 1.2.1',
+        'smart_open >= 1.7.1',
```

Comment:
this fixed in `1.7.0`, is the problem still appears in `1.7.0`?

Reply:
No, it is working for `1.7.0`. Should I change it to `1.7.0`?

Reply:
of course, this **must** be minimal supported version

Reply:
Done. :+1: